RestAPI.cs

```diff
@@ -43,12 +43,12 @@ public class RestAPI
             if (string.IsNullOrEmpty(url))
                 throw new Exception("Please use a valid WooCommerce Restful API url.");
 
-            url = url.Trim().ToLower().TrimEnd('/');

```

Comment:
I had a site like www.somesite.com/ABC/ and it would lowercase the url would return a 404. I tested this using postman as well.